### DISCUSSION TITLE
Can I combine multiple speaker weights into a single file?

### DISCUSSION THREAD

**Discussion by lextoumbourou** [voicepaw/so-vits-svc-fork#643] [Q&A] [answered]
I have a G_*.pth file with only one speaker id: `"lex": 0`.

I have a different G_*.pth with only one speaker id: `"jonno": 0`.

I want to combine them to have a single G_*.pth file with 2 speakers:
```
"lex": 0,
"jonno": 1
```

Is this possible?

Thanks for your help.

**Comment by hataori-p** [comment, marked as answer]
From what I understand about the model, I think it is not possible.
The voices should be trained jointly to make one common model.

There are speaker embeddings, which determine the speaker in the model, but I don't thing they are transferable between different models. See discussion here #282
